mob.go

```diff
@@ -494,6 +498,14 @@ func isNothingToCommit() bool {
 	return len(strings.TrimSpace(output)) == 0
 }
 
+func hasLocalCommits(currentWipBranch string) bool {
```

Comment:
can you rename `currentWipBranch` to just `branch` as this method is quite general in nature?

Reply:
Sure!